### DISCUSSION TITLE
Welcome to elmaonline-site Discussions!

### DISCUSSION THREAD

**Discussion by demiters** [elmadev/elmaonline-site#329] [General]
Apparently Discussions is now a feature on repos.
When every discussion had to be an issue before, now it should be here that discussion type issues should go instead.

Unedited placeholder text below:

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sunehs** [comment]
Cool, unfortunately can't find it as an option in the webhook so we can get notifications of these in discord.
